admin/load.php

```diff
@@ -531,6 +537,32 @@ function perflab_enqueue_modules_page_scripts() {
 	wp_enqueue_style( 'thickbox' );
 
 	wp_enqueue_script( 'plugin-install' );
+
+	// Bail early if module is not active.
+	$get_active_modules_with_standalone_plugins = perflab_get_active_modules_with_standalone_plugins();
+	if ( empty( $get_active_modules_with_standalone_plugins ) ) {
+		return;
+	}
+
+	wp_enqueue_script(
+		'perflab-module-migration-notice',
+		plugin_dir_url( __FILE__ ) . 'perflab-module-migration-notice.js',
+		array(),
+		'1.0.0',
+		array(
+			'strategy' => 'defer',
+		)
+	);
+
+	wp_localize_script(
+		'perflab-module-migration-notice',
+		'perflab_module_migration_notice',
+		array(
+			'ajaxurl'       => admin_url( 'admin-ajax.php' ),
+			'nonce'         => wp_create_nonce( 'perflab-install-activate-plugins' ),
+			'network_error' => esc_html__( 'Network response was not ok.', 'performance-lab' ),
```

Comment:
Usually I'd recommend adding `wp-i18n` as a script dependency and then use `wp.i18n.__` in the JS file instead of using `wp_localize_script` for localization.

Not a big deal though because this is just 1 string, but something to keep in mind

Reply:
The `wp-i18n` has been loaded in the setting scree, so I included the dependency for it and utilized it in JavaScript. Consequently, I removed `network_error` from the PHP.